components/apps_form/apps_form_field/apps_form_field.tsx

```diff
@@ -45,29 +40,14 @@ export interface Props {
 }
 
 export default class AppsFormField extends React.PureComponent<Props> {
-    private providers: Provider[] = [];
-
     static defaultProps = {
         listComponent: ModalSuggestionList,
     };
 
-    constructor(props: Props) {
-        super(props);
-        this.setProviders();
-    }
-
     handleSelected = (selected: AppSelectOption | UserProfile | Channel) => {
         const {name, field, onChange} = this.props;
 
-        if (field.type === AppFieldTypes.USER) {
-            const user = selected as UserProfile;
-            let selectedLabel = user.username;
-            if (this.props.teammateNameDisplay) {
-                selectedLabel = displayUsername(user, this.props.teammateNameDisplay);
-            }
-            const option = {label: selectedLabel, value: user.id};
-            onChange(name, option);
-        } else if (field.type === AppFieldTypes.CHANNEL) {
+        if (field.type === AppFieldTypes.CHANNEL) {
```

Comment:
What are the implications of removing the `AppFieldTypes.USER` check here, but keeping each of the other data sources?

Reply:
this is not needed now for users and channel as it data was adapted for `AutocompleteSelector`